### PR TITLE
DEP-803 & DEP-993: Added flag to get help form to able to deploy to single page

### PR DESCRIPTION
**Pull request by tonymyers**

### CODE REVIEW

File: app/views/GovukWrapper.scala.html [hunk 1]
```diff
@@ -41,7 +42,8 @@
         mainContent: Html = HtmlFormat.empty,
         serviceInfoContent: Html = HtmlFormat.empty,
         scriptElem: Option[Html] = None,
-        navTitle: Option[String] = None)(implicit request: Request[_], messages: play.api.i18n.Messages)
+        navTitle: Option[String] = None,
+        getHelpFlag: Boolean = false)(implicit request: Request[_], messages: play.api.i18n.Messages)
```
Comment (Burdis) [inline app/views/GovukWrapper.scala.html:46]: Is this getHelpFlag still necessary?

File: app/views/MainTemplate.scala.html [hunk 2]
```diff
@@ -34,7 +34,8 @@
     sidebarLinks: Option[Html] = None,
     contentHeader: Option[Html] = None,
     bodyClasses: Option[String] = None,
-    mainClass: Option[String] = None)(mainContent: Html)(implicit request : Request[_], messages: Messages)
+    mainClass: Option[String] = None,
+    getHelpFlag: Boolean = false)(mainContent: Html)(implicit request : Request[_], messages: Messages)
```
Comment (Burdis) [inline app/views/MainTemplate.scala.html:38]: Same as above, is this flag needed?

File: app/views/GovukWrapper.scala.html (hunk 1 above)
Comment (tonymyers) [reply app/views/GovukWrapper.scala.html]: Removed

File: app/views/MainTemplate.scala.html (hunk 2 above)
Comment (tonymyers) [reply app/views/MainTemplate.scala.html]: Removed

**Review by Burdis — APPROVED** [review]

**Pull request merged**